README.md

```diff
@@ -212,6 +224,7 @@ Cloud | Mint | Mint + Remove Admin Cred | Passthrough | Manual | Token
 --- | --- | --- | --- | --- | ---
 AWS | Y | 4.4+ | Y | 4.3+ | 4.6+ (expected)
 Azure | Y | N | Y | unknown | N
+EquinixMetal | ? | ? | ? | ?
```

Comment:
@elmiko does it make sense to add more functional support beside just documenting the format? If so, it would help to get some additional context into how CCO fits into the process and what methods might make sense for Equinix Metal to support.

Reply:
that's a good question, what were you thinking?

Reply:
EM doesn't currently have fine grained credentials, so I suspect Passthrough may make the most sense, and then as additional support is added for finer grained credentials, we can likely add support for the other methods.

Reply:
passthrough seems to make sense (i don't a ton about EM though), but as long as fits those restrictions (eg can't mint new credentials) then i would agree.

i think it makes sense to add an asterisk with a note about EM. but i don't want to create an extra maintenance burden, my main goal was to make sure we have a document about what the credentials need in the resource.

> get some additional context into how CCO fits into the process 

just to make sure i don't skip on this part. as i understand it, the CCO holds the credentials that the Machine-API bits will use. when calls the cloud provider API are made then this object will get referenced to ensure we have the proper creds.

> what methods might make sense for Equinix Metal to support

another good question, i would like to confer with the installer team to see if there is anything specific we should be highlighting here.

in general, i think proceeding with the passthrough option makes sense to me but i will gather some more information from our teams.

Reply:
@detiber i followed up with some folks internally and passthrough is fine for now. in the future it would be nice if equinix supported the more granular options, but as it sounds like that does not currently exist we should be fine passthrough.